src/components/BlogPage/BlogListItem.js

```diff
@@ -1,35 +1,64 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Typography, Link } from '@material-ui/core';
+import { Typography, Card, CardContent, CardActionArea, withStyles, Divider } from '@material-ui/core';
 
 import LinkNoStyle from '../LinkNoStyle/LinkNoStyle';
 
+const styles = theme => ({
+	container: {
+		marginTop: theme.spacing(1),
+		marginBottom: theme.spacing(1),
+		width: '100%',
```

Comment:
What do you think of doing 2 cards per row instead of making each card take up the full window width? Up to you if you want to do it though, b/c this looks super fire already.

Reply:
I do like the idea of having 2 cards per row actually, but i think our font size and appearance won't make it look as good. i think the font size needs to be smaller for the 2 cards per row to look good. i think this can be backlogged as a future update (prob in sept)